lib/oas_rails/spec/media_type.rb

```diff
@@ -1,3 +1,4 @@
+include 'erb'
```

Comment:
It should be `require 'erb'`. The tests failed because of it, apparently. And I am not sure if you need to require it here, Does not do it rails and zeitwerk?